pkg/formatters/detailsFormatter.go

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package formatters
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/galasa.dev/cli/pkg/galasaapi"
+)
+
+// -----------------------------------------------------
+// Detailed format.
+const (
+	DETAILS_FORMATTER_NAME = "details"
+	DATE_FORMAT            = "2006-01-02 15:04:05"
+)
+
+type DetailsFormatter struct {
+}
+
+func NewDetailsFormatter() RunsFormatter {
+	return new(DetailsFormatter)
+
+}
+
+func (*DetailsFormatter) GetName() string {
+	return DETAILS_FORMATTER_NAME
+}
+
+func (*DetailsFormatter) FormatRuns(runs []galasaapi.Run, apiServerUrl string) (string, error) {
+	var result string = ""
+	var err error = nil
+
+	if len(runs) < 1 {
+		return result, err
+	}
+
+	buff := strings.Builder{}
+
+	for i, run := range runs {
+		var duration string = ""
```

Comment:
Can tabulating core run details be put into a separate method ? That would isolate all the complexity of doing the tabulation ?